### ISSUE TITLE
This software creates a 'utils' subdirectory in /usr/lib/python3.11/site-packages/

### ISSUE THREAD

**Issue by evertvorster** [singularity/singularity#335]
**Describe the bug**
'utils' is a fairly generic name for a directory, and as python looks in the /usr/lib/python3.11/site-packages/
fairly early on in its path statement, this causes python software that have 'utils' as one of their subdirectories to not run properly, because it is looking in the wrong place. 

On Arch linux, at least, in the way its packaged, singularity ends up as a system-wide install, and so it's directories are visible to all other system packages, and with a common name like 'utils' it's causing issues in other python software if they have a 'utils' subdirectory. 

As a favor, would you please consider renaming that directory to something less likely to cause a conflict, like singularity-utils? Or move it into singularity's directory so it's out of the default python path?

About my system:
```
Operating System: Arch Linux 
KDE Plasma Version: 5.27.9
KDE Frameworks Version: 5.112.0
Qt Version: 5.15.11
Kernel Version: 6.6.1-arch1-1 (64-bit)
Graphics Platform: Wayland
Processors: 32 × 13th Gen Intel® Core™ i9-13900HX
Memory: 62.5 GiB of RAM
Graphics Processor: Mesa Intel® Graphics
Manufacturer: TUXEDO
Product Name: PD5x_7xSNC_SND_SNE
System Version: Not Applicable
```

**Comment by nthykier** [comment]
Evert Vorster:
> **Describe the bug**
> 'utils' is a fairly generic name for a directory, and as python looks in the /usr/lib/python3.11/site-packages/
> fairly early on in its path statement, this causes python software that have 'utils' as one of their subdirectories to not run properly, because it is looking in the wrong place.
> 
> On Arch linux, at least, in the way its packaged, singularity ends up as a system-wide install, and so it's directories are visible to all other system packages, and with a common name like 'utils' it's causing issues in other python software if they have a 'utils' subdirectory.
> 
> As a favor, would you please consider renaming that directory to something less likely to cause a conflict, like singularity-utils? Or move it into singularity's directory so it's out of the default python path?
> 
> [...]

Hi,

The `utils` folder is not supposed to be installed in the first place 
(not to be confused with the `singularity.utils` package which should 
and is installed - but that one is namespaced and should not cause issues).

Happy to accept a patch for `setup.py` to fix this problem.

Best regards,
Niels

**Comment by evertvorster** [comment]
Hi there!

Thanks for the quick reply. I'll poke at it a little in my spare time. 

Kind regards,
-Evert-

**Comment by evertvorster** [comment]
Made a pull request. 
Admittedly, this is not my code, but was suggested in this bug report:
https://bugs.archlinux.org/task/80296

**Comment by nthykier** [comment]
Closed via #336
